docs/src/snap/howto/refresh-certs.md

```diff
@@ -0,0 +1,98 @@
+# Refreshing Kubernetes Certificates
+
+Keeping your {{product}} cluster secure and functional requires
+regularly refreshing its certificates. Certificates in Kubernetes ensure
+secure communication between the various components of the cluster. If
+these certificates expire, it can lead to communication failures, disrupted
+services, and potential security risks. This how-to will walk you through
+the steps to refresh the certificates for both control plane and worker
+nodes in your {{product}} cluster.
+
+## What you will need
+- A running {{product}} cluster
+
+```{note} To refresh the certificates in your cluster, make sure it was
+initially set up with self-signed certificates or with your own CA certificates
+and keys during the bootstrap process.
```

Comment:
I don't think we currently can rotate certificates that are signed by external CA, can we? 
In the proposal, this was out of scope: https://github.com/canonical/k8s-snap/blob/main/docs/proposals/002-refresh-certificates.md#external-ca-certificate

Reply:
Technically, we can. In the bootstrap configuration, you can provide all the [certificates](https://github.com/canonical/k8s-snap-api/blob/main/api/v1/type_bootstrap_config.go#L27-L32) for the cluster, including the CA cert and CA key, which allow the cluster to manage its own certificates. This is possible if the user provides all the necessary custom certificates. However, if it causes more confusion, I’ll remove that part and leave only the self-signed note.

Reply:
True, I was messing that up with CA rotation. It's fine to keep it